NRC.nl.js

```diff
@@ -0,0 +1,157 @@
+{
+	"translatorID": "8e708ceb-ce42-446f-bffd-21e2bdeb0742",
+	"label": "NRC.nl",
+	"creator": "Martijn Staal",
+	"target": "^https?://www.nrc.nl",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2023-06-15 13:47:40"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+	
+	Copyright © 2023 Martijn Staal
+	
+	This file is part of Zotero.
+	
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+	
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes('/nieuws/')) {
+		return "newspaperArticle";
+	}
+
+	return false;
+}
+
+function doWeb(doc, url) {
+	return scrape(doc, url)
+}
+
+function scrape(doc, url) {
+	let item = new Zotero.Item("newspaperArticle");
```

Comment:
You'll want to start with the embedded metadata translator, which gets you most of the below (title, language, date, description, rights, url, snapshot) already. Use the template available in the translator editor